packages/components/src/IconsProvider/IconsProvider.component.js

```diff
@@ -14,14 +19,14 @@ import talendIcons from '@talend/icons/dist/react';
 function IconsProvider({ defaultIcons, icons }) {
 	const iconset = Object.assign({}, defaultIcons, icons);
 	const ids = Object.keys(iconset);
-	const style = { display: 'none' };
 	return (
-		<svg xmlns="http://www.w3.org/2000/svg" focusable="false" style={style}>
+		<svg xmlns="http://www.w3.org/2000/svg" focusable="false" className={theme.hide}>
 			{ids.map((id, index) => (
 				<symbol key={index} id={id}>
 					{iconset[id]}
 				</symbol>
 			))}
+			{Object.keys(filters).map((id, index) => addKeyValue(filters[id], index))}
```

Comment:
Why don't you use the same thing as icons ?
```
{Object.keys(filters).map((id, index) => (
				<symbol key={`filter-${index}`} id={id}>
					{filters[id]}
				</symbol>
			))}
```

Reply:
filter doesn't use symbol. We have just to add the filter in the svg container.

Reply:
Right, with symbol it doesn't work on FF.

Without `symbol` : 
```
{Object.keys(filters).map((id, index) => (
	<svg key={`svg-filter-${index}`} id={id}>
		{filters[id]}
	</svg>
))}
```

This works on Chrome, FF, Safari, and it avoid the `addKeyValue` function that is quite hard to understand